src/sections/Projects/Sistent/components/content.js

```diff
@@ -1,4 +1,12 @@
 const componentsData = [
+  {
+    id: 0,
```

Comment:
no need to add it to first, there is sort functionality already there which will sort based on alphabetical order